res/values/strings.xml

```diff
@@ -21,4 +21,6 @@
     <string name="quote_download_failed">Failed to download quote</string>
     <string name="quote_download_success">Quote downloaded successfully</string>
     <string name="module_vnaas_name">Visual Novel as a Service</string>
+    <string name="module_goodreadsQotd_name">Goodreads QOTD RSS</string>
```

Comment:
Please follow the naming conventions:
`module_goodreadsQotd_name` -> `module_goodreads_name`
`module_Hitokoto_name` -> `module_hikoto_name`